### DISCUSSION TITLE
whole brain reference

### DISCUSSION THREAD

**Discussion by nikparastf** [kschan0214/sepia#27] [Q&A]
Greetings and Regards.
I am writing an article, I cited sepia . But I did not find a reference to the whole brain referencing in your article.
Thank you for sending me the related reference.
Thanks

**Comment by kschan0214** [comment]
Hi,

I think the following paper could be related to what you are looking for:

Feng, X., Deistung, A., Reichenbach, J.R., 2018. Quantitative susceptibility mapping (QSM) and R2 * in the human brain at 3T Evaluation of intra-scanner repeatability. Zeitschrift Für Medizinische Physik 28, 36–48. https://doi.org/10.1016/j.zemedi.2017.05.003
